### PR TITLE
Docs: Resize feeds image

### LINKED ISSUE #1478: Docs: The rss_feed.png image should be resized/scaled to fit the screen
Hello!

On page https://www.getzola.org/documentation/templates/feeds/:

zola

https://www.getzola.org/documentation/templates/feeds/rss_feed.png needs to be scaled/resized horizontally to fit the screen.

Thanks!

### PR DESCRIPTION
**Pull request by RhnSharma**
Fixes #1478 
Hi @Keats @rootkea, how does this look?
Let me know if this needs any changes.
Thanks

### CODE REVIEW

File: docs/sass/_normalize.scss [hunk 1]
```diff
@@ -219,6 +220,8 @@ audio:not([controls]) {
 
 img {
   border-style: none;
+  max-width: 100%;
+  height: auto;
```
Comment (Keats) [inline docs/sass/_normalize.scss]: Please add the rule in the main scss file, this is a vendored file

Comment (RhnSharma) [reply docs/sass/_normalize.scss]: @Keats Okay, Could you please tell me which is the main file? Is _docs.scss the main file? Or do I have to make changes in the site.scss file?

Comment (Keats) [reply docs/sass/_normalize.scss]: Put it in this file: https://github.com/getzola/zola/blob/master/docs/sass/_base.scss

**Comment by RhnSharma** [comment]
@Keats I moved the changes to _base.scss file. Please take a look at it.
Thanks

**Comment by Keats** [comment]
Can you turn off your editor CSS formatter? Ideally only https://github.com/getzola/zola/pull/1485/files#diff-97ddea12e5ac7d077e685bf642c37b098f763aa24fbbae64387522bdcf3c1d9aR23-R27 would be in the commit

**Comment by RhnSharma** [comment]
@Keats Okay, I'll fix the formatting.

**Review by Keats — APPROVED** [review]

**Pull request merged**